dashboard/test/controllers/lessons_controller_test.rb

```diff
@@ -9,17 +9,13 @@ class LessonsControllerTest < ActionController::TestCase
     # stub writes so that we dont actually make updates to filesystem
     File.stubs(:write)
 
-    @script = create :script, name: 'unit-1', is_migrated: true
+    @script = create :script, name: 'unit-1'
     lesson_group = create :lesson_group, script: @script
     @lesson = create(
       :lesson,
-      script_id: @script.id,
       lesson_group: lesson_group,
       name: 'lesson display name',
-      absolute_position: 1,
-      relative_position: 1,
       has_lesson_plan: true,
```

Comment:
Should has_lesson_plan have a default value too like lockable?

Reply:
it does have a default value: https://github.com/code-dot-org/code-dot-org/blob/330708bcf354d95b3ed0486132b4028340afc3f0/dashboard/test/factories/factories.rb#L919